docker/workspace-build/Dockerfile

```
@@ -84,6 +84,11 @@ RUN cd "${ANGEL_WORKSPACE_DIR}" \
       pip3 install --no-deps --user -r /dev/stdin \
  && pip uninstall -y setuptools-scm
 
+# Build the angel_system python package
+COPY ./angel_system "${ANGEL_WORKSPACE_DIR}"/angel_system
+RUN cd "${ANGEL_WORKSPACE_DIR}" \
+ && pip install .
```

We should add the following options:
```suggestion
 && pip install --no-deps --user -e .
```

See the note about updating poetry-code in the build-system section.